src/.copier/_generate_apis.py.jinja

```diff
@@ -0,0 +1,16 @@
+from pathlib import Path
+from os import path
+from copier import run_auto
+copier_base_dir = Path.cwd()
+
+apis = {{apis}}
+app_name = "{{app_name}}"
+potential_answers_file: Path = Path.cwd() / "copier-{{api}}.yaml"
+# a programmatic way to get "dst_path" argument's value that user provided via the command line...
+
+for api in apis:
+    if not path.exists(copier_base_dir/"apps"/api):
+        answers_file = potential_answers_file if potential_answers_file.exists() else None
```

Comment:
Why introduce the `potential_answers_file`?

In the previous shell script it was simply passed as `.copier-{{api}}.yaml` with not additional checks